frontend/components/adopter/Landing/common.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const PICTURE_ONE =
```

Comment:
hey just want to check, is there some access needed to be able to see the pictures?
on my end when i run `npm run dev` I don't see all the pics at all
![image](https://user-images.githubusercontent.com/28668944/139576136-8e1b6ee4-a85f-4680-96c2-c7d7e544bc50.png)